plugins/vee-validate.js

```diff
@@ -21,3 +21,15 @@ extend("min_value", {
     return `You must be ${args.min} years old or older `;
   }
 });
+
+extend("size", {
+  ...size,
+  message: (value, args) => {
+    return `File size should be less than ${args.size/1000} MB`;
+  }
+});
+
+extend("ext", {
+  ...ext,
+  message: "The image format must be jpg, jpeg, png, or bmp"
```

Comment:
Mestinya message `must be jpg, jpeg, png, or bmp` tidak hardcoded ya? Karena itu kan di-supply pada atribut `<element rules="ext:<ekstensi_1>, <ekstensi_2>">`.

Baiknya dibuat rule khusus aja, misalnya `ext-image`.

Reply:
@adrianpdm iya kang, kemaren udah dicoba yang dinamis nya. cuma passingan dari atributnya bentuknya object bukan array, jadi bingung kemaren nampilinnya gimana 😅